.github/workflows/deployment-pr.yml

```diff
@@ -1,152 +1,28 @@
-          echo "Will uninstall: $TO_UNINSTALL"
-          echo "list=$TO_UNINSTALL" | tr -d "\n" >> $GITHUB_OUTPUT
-          echo "\n" >> $GITHUB_OUTPUT
-          echo "never_uninstall=$never_uninstall" | tr -d "\n" >> $GITHUB_OUTPUT
-
-      - name: List PRs without up-to-date deployment
-        id: to-deploy
-        env:
-          deployments: ${{ steps.current-deployments.outputs.list }}
-          prs: ${{ steps.deployment-candidates.outputs.list }}
-        run: |
-          TO_INSTALL=$(jq --null-input --argjson prs '${{ env.prs }}' --argjson deployments '${{ env.deployments }}' \
-            '$prs|map(select([{name:.name,sha:.sha}]|inside($deployments)|not))')
-          echo "Will install the following candidates, because they either aren't deployed or their deployment is out of date: $TO_INSTALL"
-          echo "list=$TO_INSTALL" | tr -d "\n" >> $GITHUB_OUTPUT
-
-  uninstall-old-deployment:
-    name: Uninstall old deployment
-    runs-on: ubuntu-latest
-    needs:
-      - find-prs-to-deploy
-    if: fromJSON(needs.find-prs-to-deploy.outputs.to-uninstall)[0] != null
-    strategy:
-      fail-fast: false
-      matrix:
-        deployment-name: ${{ fromJSON(needs.find-prs-to-deploy.outputs.to-uninstall) }}
-    steps:
-
-      - name: Uninstall helm release
-        run: |
-          # Setup authentication
-          mkdir ~/.kube && echo '${{ secrets.KUBECONFIG }}' > ~/.kube/config && chmod go-r ~/.kube/config
-          # Uninstall the release
-          helm delete ecamp3-${{ matrix.deployment-name }}
-
-      - name: Remove GitHub deployment
-        uses: bobheadxi/deployments@v1.4.0
-        with:
-          step: deactivate-env
-          token: ${{ secrets.REPO_ACCESS_TOKEN }}
-          env: ${{ matrix.deployment-name }}
-
-      - name: Find former PR number based on deployment name
-        id: pr-number
-        env:
-          name: ${{ matrix.deployment-name }}
-          regex: '^pr([0-9]+)$'
-        run: |
-          [[ $name =~ $regex ]]
-          echo number=${BASH_REMATCH[1]} >> $GITHUB_OUTPUT
-
-      - name: Comment about uninstallation on PR
-        uses: thollander/actions-comment-pull-request@v2
-        if: steps.pr-number.outputs.number
-        with:
-          pr_number: ${{ steps.pr-number.outputs.number }}
-          message: |
-            ### <span aria-hidden="true">⛔</span> Feature branch deployment currently inactive.
-
-
-            If the PR is still open, you can add the `deploy!` label to this PR to trigger a feature branch deployment.
-          comment_tag: feature-branch-deployment-status
-          create_if_not_exists: false
-
   build-and-push:
-    name: Build and push docker images  
-    needs: find-prs-to-deploy
-    if: fromJSON(needs.find-prs-to-deploy.outputs.to-deploy)[0] != null
+    if: contains(github.event.pull_request.labels.*.name, 'deploy!')
```

Comment:
cool, you can have a wildcard in the json path

Reply:
Yep, pretty cool
https://docs.github.com/en/actions/learn-github-actions/expressions#example-using-an-object-filter